README.md

```diff
@@ -1 +1,9 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
```

Comment:
Kindly use the Microverse README template for the [README file](https://github.com/microverseinc/readme-template) with its styling and add the same information you added to that template.

Reply:
May I have to change the README.md file in the main branch?

Reply:
No, do all changes in the branch and you can merge it to main after approval.